src/api/pastel-rpc/network-stats/network-stats.ts

```diff
@@ -0,0 +1,480 @@
+import {
+  IBlockInfo,
+  IBlockSubsidy,
+  IChainTips,
+  IListTransactions,
+  IListUnspent,
+  IMempoolInfo,
+  IMiningInfo,
+  INetTotals,
+  INetworkInfo,
+  IRawTransaction,
+  IReceivedByAddress,
+  ITotalBalance,
+  ITransactionInfo,
+  ITxoutsetInfo,
+  IWalletInfo,
+} from '../../../type'
+import { rpc, TRPCConfig } from '../rpc'
+
+interface IStatistic {
+  hashrate: string
+  difficulty: string
+}
+
+export async function getStatisticInfo(
+  config: TRPCConfig,
+): Promise<IStatistic> {
+  try {
+    const [hashrate, difficulty] = await Promise.all([
+      getNetworkHashps(config),
+      getDifficulty(config),
+    ])
+    return {
+      hashrate: hashrate,
+      difficulty: difficulty,
+    }
+  } catch (error) {
+    throw new Error(`network-stats getStatisticInfo error: ${error.message}`)
+  }
+}
+
+export async function getNetworkHashps(config: TRPCConfig): Promise<string> {
+  const { result } = await rpc('getnetworkhashps', [], config)
```

Comment:
Could we please change it to `rpc<T>(...)`, where T is the response type of this RPC call for all functions in this file? Make sure you are returning the right type though, because I know that `getnetworkhashps` returns this object as response:
```
{
  "result": 14523646,
  "error": null,
  "id": "curltest"
}
```
Notice that "result" here is a number, not a string

Reply:
To properly check what is the return type of every call, you can do the following:
1. Download Insimnia (if you don't have it yet): https://insomnia.rest/
2. Import my config (I created the first 3 methods): https://gist.github.com/br4in3x/a16e37f0040829c3db5190a0b2198ce3
3. Run `pasteld`:
  3.1. cd into the folder with binaries: `cd static/bin/<your system>`
  3.2. run pasteld rpc server: `.\pasteld-win.exe --rpcport=9932 --rpcuser=insomnia --rpcpassword=insomnia`
4. Execute any call from insomnia, you will get something like this:
![image](https://user-images.githubusercontent.com/1525725/115604728-b4217480-a2af-11eb-911b-b959bf37e0de.png)